### DISCUSSION TITLE
Monitoring GPU Temp with NootRX running

### DISCUSSION THREAD

**Discussion by menthorz** [ChefKissInc/NootRX#96] [Q&A] [answered]
How can show the temp with the kext running on current system?

**Comment by VisualEhrmanntraut** [comment, marked as answer]
https://github.com/ChefKissInc/SMCRadeonSensors

**Reply by menthorz** [reply]
Tks for reply.
When I applied this kext, kp, not boot up.

**Reply by menthorz** [reply]
I fixed. Just remove WhateverGreen from the EFI folder. Tks for the time and help.
